### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by Xander-Lambright** [tofodroid/mimi-mod#99]
**Question**
I'm unsure how to properly load a soundfont. Whenever I direct the config file to the .sf2 file, MineCraft crashes on startup. It also crashes no matter what I put on that line however. I'm very new to MIDI and so I don't know if I am downloading the wrong kind of file or putting it in the wrong place. Appreciate any guidance :).

Here's the path that I used : "C:\Users\Yellow768\AppData\Roaming\.minecraft\Forge Versions\IslesOfBeicula(1.18.2)\The_Mini_Cheapy_Orch_GM.sf2"

**Environment (if applicable):**
 - Windows 10
 - Mod Version: 1.18.2-3.0.3
 - Minecraft Version: 1.18.2
 - Forge Version: 1.18.2-40.2.10
<details>
<summary>Crash Report</summary>

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 1/26/24, 5:28 PM
Description: Rendering overlay

net.minecraftforge.fml.config.ConfigFileTypeHandler$ConfigLoadingException: Failed loading config file mimi-client.toml of type CLIENT for modid mimi
	at net.minecraftforge.fml.config.ConfigFileTypeHandler.lambda$reader$1(ConfigFileTypeHandler.java:47) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:60) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:50) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {}
	at java.util.Collections$SynchronizedCollection.forEach(Collections.java:2131) ~[?:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:50) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.core.ModStateProvider.lambda$new$1(ModStateProvider.java:33) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading}
	at net.minecraftforge.fml.DistExecutor.unsafeRunWhenOn(DistExecutor.java:111) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.core.ModStateProvider.lambda$new$3(ModStateProvider.java:33) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading}
	at net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$20(ModLoader.java:186) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at java.util.Optional.ifPresent(Optional.java:178) ~[?:?] {}
	at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:186) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.ModLoader.lambda$loadMods$14(ModLoader.java:170) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {}
	at net.minecraftforge.fml.ModLoader.loadMods(ModLoader.java:170) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$startModLoading$5(ClientModLoader.java:121) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:112) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.startModLoading(ClientModLoader.java:121) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$onResourceReload$2(ClientModLoader.java:103) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:112) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}
Caused by: com.electronwill.nightconfig.core.io.ParsingException: Invalid value: sers\Yel
	at com.electronwill.nightconfig.core.io.Utils.parseLong(Utils.java:67) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.core.io.Utils.parseInt(Utils.java:83) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.toml.StringParser.escape(StringParser.java:134) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.StringParser.parseBasic(StringParser.java:25) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.ValueParser.parse(ValueParser.java:45) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.ValueParser.parse(ValueParser.java:69) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.TableParser.parseNormal(TableParser.java:57) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:88) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:37) ~[toml-3.6.4.jar%238!/:?] {}
	at com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:113) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:219) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:202) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.core.file.WriteSyncFileConfig.load(WriteSyncFileConfig.java:73) ~[core-3.6.4.jar%237!/:?] {}
	at com.electronwill.nightconfig.core.file.AutosaveCommentedFileConfig.load(AutosaveCommentedFileConfig.java:85) ~[core-3.6.4.jar%237!/:?] {}
	at net.minecraftforge.fml.config.ConfigFileTypeHandler.lambda$reader$1(ConfigFileTypeHandler.java:43) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	... 26 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraftforge.fml.config.ConfigFileTypeHandler.lambda$reader$1(ConfigFileTypeHandler.java:47) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:60) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:50) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {}
	at java.util.Collections$SynchronizedCollection.forEach(Collections.java:2131) ~[?:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:50) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.core.ModStateProvider.lambda$new$1(ModStateProvider.java:33) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading}
	at net.minecraftforge.fml.DistExecutor.unsafeRunWhenOn(DistExecutor.java:111) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
	at net.minecraftforge.fml.core.ModStateProvider.lambda$new$3(ModStateProvider.java:33) ~[forge-1.18.2-40.2.10-universal.jar%2361!/:?] {re:classloading}
	at net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$20(ModLoader.java:186) ~[fmlcore-1.18.2-40.2.10.jar%2357!/:?] {}
-- Overlay render details --
Details:
	Overlay name: net.minecraft.client.gui.screens.LoadingOverlay
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:882) ~[client-1.18.2-20220404.173914-srg.jar%2356!/:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1046) ~[client-1.18.2-20220404.173914-srg.jar%2356!/:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665) ~[client-1.18.2-20220404.173914-srg.jar%2356!/:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%2356!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.2.10.jar%2318!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: No
	Packs: Default, Mod Resources, Isles of Beicula
	Recovery: Yes
	Recovery reason: java.util.concurrent.CompletionException: net.minecraftforge.fml.config.ConfigFileTypeHandler$ConfigLoadingException: Failed loading config file mimi-client.toml of type CLIENT for modid mimi
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: net.minecraftforge.fml.config.ConfigFileTypeHandler$ConfigLoadingException: Failed loading config file mimi-client.toml of type CLIENT for modid mimi
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.config.ConfigFileTypeHandler.lambda$reader$1(ConfigFileTypeHandler.java:47)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:60)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:50)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at java.base/java.util.Collections$SynchronizedCollection.forEach(Collections.java:2131)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:50)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.fml.core.ModStateProvider.lambda$new$1(ModStateProvider.java:33)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.DistExecutor.unsafeRunWhenOn(DistExecutor.java:111)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.fml.core.ModStateProvider.lambda$new$3(ModStateProvider.java:33)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$20(ModLoader.java:186)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:186)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.ModLoader.lambda$loadMods$14(ModLoader.java:170)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.ModLoader.loadMods(ModLoader.java:170)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.client.loading.ClientModLoader.lambda$startModLoading$5(ClientModLoader.java:121)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:112)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.client.loading.ClientModLoader.startModLoading(ClientModLoader.java:121)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.client.loading.ClientModLoader.lambda$onResourceReload$2(ClientModLoader.java:103)
	at TRANSFORMER/forge@40.2.10/net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:112)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	... 6 more
Caused by: com.electronwill.nightconfig.core.io.ParsingException: Invalid value: sers\Yel
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.io.Utils.parseLong(Utils.java:67)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.io.Utils.parseInt(Utils.java:83)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.StringParser.escape(StringParser.java:134)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.StringParser.parseBasic(StringParser.java:25)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.ValueParser.parse(ValueParser.java:45)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.ValueParser.parse(ValueParser.java:69)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.TableParser.parseNormal(TableParser.java:57)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:88)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.toml@3.6.4/com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:37)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:113)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:219)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.io.ConfigParser.parse(ConfigParser.java:202)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.file.WriteSyncFileConfig.load(WriteSyncFileConfig.java:73)
	at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightconfig.core.file.AutosaveCommentedFileConfig.load(AutosaveCommentedFileConfig.java:85)
	at LAYER PLUGIN/fmlcore@1.18.2-40.2.10/net.minecraftforge.fml.config.ConfigFileTypeHandler.lambda$reader$1(ConfigFileTypeHandler.java:43)
	... 26 more

-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 2438503112 bytes (2325 MiB) / 3758096384 bytes (3584 MiB) up to 12884901888 bytes (12288 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 2600 Six-Core Processor            
	Identifier: AuthenticAMD Family 23 Model 8 Stepping 2
	Microarchitecture: Zen+
	Frequency (GHz): 3.39
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1660 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2182
	Graphics card #0 versionInfo: DriverVersion=31.0.15.4601
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 16384.00
	Memory slot #2 clockSpeed (GHz): 3.20
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 16384.00
	Memory slot #3 clockSpeed (GHz): 3.20
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 75212.40
	Virtual memory used (MB): 41342.83
	Swap memory total (MB): 9728.00
	Swap memory used (MB): 1197.82
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx12G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: 1.18.2-forge-40.2.10
	Backend library: LWJGL version 3.2.2 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 GL version 3.2.0 NVIDIA 546.01, NVIDIA Corporation
	Window size: 854x480
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: English (US)
	CPU: 12x AMD Ryzen 5 2600 Six-Core Processor 
	ModLauncher: 9.1.3+9.1.3+main.9b69c82a
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		 mixin PLUGINSERVICE 
		 eventbus PLUGINSERVICE 
		 slf4jfixer PLUGINSERVICE 
		 object_holder_definalize PLUGINSERVICE 
		 runtime_enum_extender PLUGINSERVICE 
		 capability_token_subclass PLUGINSERVICE 
		 accesstransformer PLUGINSERVICE 
		 runtimedistcleaner PLUGINSERVICE 
		 mixin TRANSFORMATIONSERVICE 
		 fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |COMMON_SET|Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		forge-1.18.2-40.2.10-universal.jar                |Forge                         |forge                         |40.2.10             |COMMON_SET|Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		Patchouli-1.18.2-71.1.jar                         |Patchouli                     |patchouli                     |1.18.2-71.1         |COMMON_SET|Manifest: NOSIGNATURE
		mimimod-1.18.2-3.0.3.jar                          |MIMI                          |mimi                          |1.18.2-3.0.3        |COMMON_SET|Manifest: NOSIGNATURE
	Crash Report UUID: a05c4971-4907-4d1e-9024-2f8cd9024d22
	FML: 40.2
	Forge: net.minecraftforge:40.2.10

</details>

**Comment by Merancapeman** [comment]
Since the author doesn't seem to care, I found it's best to use the in-game config to set the file path of the soundfont. Make sure the sf file isn't too big either. I tried a 4GB one (I know that's ridiculous for a sf2 file) and it wouldn't do it. Around 1GB worked for me.

**Comment by tofodroid** [comment]
Hello! Apologies for the super late reply. Life got super crazy for awhile and I had pretty much no time for the mod. @Merancapeman is definitely right on with the file size and where to put the file. Technically anywhere should work, but the shorter the path is the easier it is to make sure it's right! I'd also recommend trying a path that doesn't include any spaces because sometimes Java and spaces in file paths just don't get along.

Please let me know if you have any other issues and thanks @Merancapeman for helping out!

**Comment by tofodroid** [comment]
Hello again - I've added a Wiki page for loading Soundfonts which should hopefully help clarify this process. I took a closer look at the log file you attached and it appears that the issue may be that your filepath is using backslashes (\). Due to the way that Minecraft parses config files if your path contains backslashes they will either need to be doubled (\\) or replaced with forward slashes (/). Let me know if that helps and if you still have any issues!

https://github.com/tofodroid/mimi-mod/wiki/Settings#synth-soundfont
